content/200-concepts/050-overview/300-prisma-in-your-stack/03-is-prisma-an-orm.mdx

```diff
@@ -292,68 +307,138 @@ generator client {
 }
 ```
 
-2. Run `prisma introspect`
-3. Run `prisma generate`
+2. Run `prisma introspect` which will populate the Prisma schema with models derived from your database schema.
+3. (Optional) Customize [field and model mappings](<(https://www.prisma.io/docs/concepts/components/prisma-schema/data-model#mapping-model-names-to-tables)>) between Prisma Client and the database.
+4. Run `prisma generate`.
 
-Prisma will generate Prisma Client inside the `node_modules` folder, from which it can be imported in your application. For more extensive usage documentation, see the [Prisma Client API](../../components/prisma-client) docs.
+Prisma will generate Prisma Client inside the `node_modules` folder, from which it can be imported in your application. For more extensive usage documentation, see the [Prisma Client API](../../reference/tools-and-interfaces/prisma-client/api) docs.
 
 To summarise, Prisma Client can be integrated into projects with an existing database and tooling as part of a parallel adoption strategy. New projects will use a different workflow detailed next.
 
 #### Workflow for new projects
 
 Prisma is different from ORMs in terms of the workflows it supports. A closer look at the steps necessary to create and change a new database schema is useful for understanding Prisma Migrate.
 
-Prisma Migrate is a CLI for declarative data modeling & migrations. Unlike most migration tools that come as part of an ORM, you only need to describe the current schema, instead of the operations to move from one state to another. Prisma Migrate infers the operations and carries out the migration for you.
+Prisma Migrate is a CLI for declarative data modeling & migrations. Unlike most migration tools that come as part of an ORM, you only need to describe the current schema, instead of the operations to move from one state to another. Prisma Migrate infers the operations, generates the SQL and carries out the migration for you.
 
-This example demonstrates using Prisma in a new project with a new database schema similar to the Sequelize example above:
+This example demonstrates using Prisma in a new project with a new database schema similar to the blog example above:
 
 1. Create the Prisma schema:
 
 ```prisma
 // schema.prisma
 datasource db {
-  provider = "sqlite"
-  url      = "file:data.db"
+  provider = "postgres"
```

Comment:
Default is still `postgresql`: https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference/#datasource